docs/sources/installation/local.md

```diff
@@ -11,8 +11,23 @@ In order to log events with Grafana Loki, download and install both Promtail and
 
 The configuration specifies running Loki as a single binary.
 
-## Install
+## Install using APT or RPM package managers
 
+1. Add Granafa's [APT](https://apt.grafana.com/) or [RPM](https://rpm.grafana.com/)
```

Comment:
I'm just going to state for the record that I hate that RPM stands for RPM Package Manager.   UGH.

```suggestion
1. Add Granafa's Advanced Package Tool [APT](https://apt.grafana.com/) or RPM Package Manager [RPM](https://rpm.grafana.com/)
```